server/game/cards/10-TEL/RightHandOfTheEmperor.js

```diff
@@ -0,0 +1,37 @@
+import { Locations, Players, CardTypes, TargetModes } from '../../Constants.js';
+
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+
+class RightHandOfTheEmperor extends DrawCard {
+    setupCardAbilities() {
+        this.persistentEffect({
+            condition: context => context.player.honor > context.player.opponent.honor,
+            location: Locations.ConflictDiscardPile,
+            effect: AbilityDsl.effects.canPlayFromOwn(Locations.ConflictDiscardPile, [this])
+        });
+        this.action({
+            title: 'Ready characters',
+            target: {
+                mode: TargetModes.MaxStat,
+                activePromptTitle: 'Choose characters',
+                cardStat: card => card.getCost(),
+                maxStat: () => 6,
+                numCards: 0,
+                cardType: CardTypes.Character,
+                controller: Players.Self,
+                cardCondition: card => card.hasTrait('bushi') && card.bowed,
+                gameAction: AbilityDsl.actions.sequential([AbilityDsl.actions.ready(),
```

Comment:
this should be `multiple` - also can you format in line with other cards? :)

Reply:
Switched to multiple.  I think the formatting is better now, looked at a few examples of other cards.